etc/mpv.profile

```diff
@@ -16,11 +16,15 @@ include /etc/firejail/disable-passwdmgr.inc
 
 caps.drop all
 netfilter
+nogroups
 nonewprivs
 noroot
 protocol unix,inet,inet6
 seccomp
+tracelog
 
 # to test
+ipc-namespace
```

Comment:
Looks good but let's comment out `ipc-namespace` please. We've had some breakages on other systems.